kafka/consumer/Dockerfile

```diff
@@ -3,14 +3,15 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
-FROM golang:1.21 as builder
-WORKDIR /app
-COPY go.* ./
-RUN go mod download
-COPY main.go ./
-RUN CGO_ENABLED=0 GOOS=linux go build -mod=readonly  -v -o consume
+FROM golang:1.22 as builder
+ENV PROJECT_DIR=/go/src/github.com/microsoft
```

Comment:
This should probably be ARG instead of ENV.  ENV will be persisted and passed to processes that run.